cloudevents-sdk-reqwest/src/client_response.rs

```diff
@@ -10,6 +11,24 @@ use reqwest::header::{HeaderMap, HeaderName};
 use reqwest::Response;
 use std::convert::TryFrom;
 
+/// Extention Trait for [`Response`]
+#[async_trait]
+pub trait ResponseExt {
+    async fn response(self) -> Result<Event>;
```

Comment:
I would prefer to call this method `to_event()`

Reply:
done